app/views/devise/passwords/edit.html.haml

```diff
@@ -1,25 +1,22 @@
 = content_for :page_title, flush: true do
   = t('.page_title')
 
-= render partial: 'errors/error_summary', locals:{ ep: resource, error_summary: t('.error_summary')}
-
 = render partial: 'layouts/header', locals: {page_heading: t('.page_heading')}
 
-= simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f|
-  .form-group
-  = f.hidden_field :reset_password_token
-  = f.full_error :reset_password_token
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    = form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }, builder: GOVUKDesignSystemFormBuilder::FormBuilder) do |f|
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3698/files#r565935858)